application/main.tf

```diff
@@ -108,3 +108,23 @@ data "aws_ami" "centos_wls" {
     values = ["ebs"]
   }
 }
+
+data "aws_ami" "centos_oracle_db" {
+  owners      = ["895523100917"]
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["HMPPS Delius-Core OracleDB master *"] 
```

Comment:
Won't work until we're merged into master

Pleas use `HMPPS Delius-Core OracleDB *` for now